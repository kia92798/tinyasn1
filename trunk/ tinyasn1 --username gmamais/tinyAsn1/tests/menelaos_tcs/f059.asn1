F059 DEFINITIONS AUTOMATIC TAGS	EXTENSIBILITY IMPLIED ::= BEGIN
ShortINTEGER ::= INTEGER (0..6)
Citizen ::= SEQUENCE { country ENUMERATED {france, greece, italy, germany},
	    yearOfBirth	INTEGER,
	    married BOOLEAN DEFAULT TRUE,
	    nOfChildren ShortINTEGER}

	    GreekCitizenWithIndicationOfMarriage ::= Citizen (WITH COMPONENTS {...,
					country (greece),
					married PRESENT}) -- this should fail as it shouldn't make
	                                                  -- sense to specify a DEFAULT constraint
							  -- with PRESENT

	    aGreek GreekCitizenWithIndicationOfMarriage ::= {greece, 1960,2} 
END
