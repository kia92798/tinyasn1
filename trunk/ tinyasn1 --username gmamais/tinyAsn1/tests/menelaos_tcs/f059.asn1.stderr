Error in line : 14. Expecting SEQUENCE variable
